src/.sink.d.ts

```diff
@@ -1,3 +1,4 @@
 // These packages don't have any TS types so we need to declare a module in order to use them.
+declare module '@readme/eslint-config';
```

Comment:
Since this doesn't need to get loaded for the whole app you might be able to move this line into the top of the new script you made. That might work?

Reply:
weird, I thought I needed this but I think I'm able to remove this line entirely and not re-add it anywhere. let me try that

Reply:
Might also want to exclude that script from being added into `dist/` if `npm run build` is adding it there now.

Reply:
looks like we already exclude the `bin` directory:

https://github.com/readmeio/rdme/blob/d2b96d4474003e04b8a7543bfb6de075ef02aa54/tsconfig.json#L17

Reply:
cool looks like tests/build are passing without this line! weird